src/collective/listingviews/browser/views/templates/layout.pt

```diff
@@ -14,6 +14,9 @@
 
         <div class="listingviews" tal:define="adapter nocall:context/@@listing_view_adapter;
             batch folderContents | adapter/retrieve_listing_items;">
+
+             <p id="result_count" tal:condition="python:adapter.display_count">Showing <span tal:content="adapter/number_of_items" /> items</p>
```

Comment:
no i18n

Reply:
What would i18n look like here?

Doesn't seem to be any internationalisation on the package, so perhaps this is more something for a separate issue/PR?

If it's just a case of adding the i18n schema declaration and `i18n:domain="collective.listingviews"` to the paragraph then easily done in this PR.